docs/api-docs/store-management/shipping/shipping-provider-api.md

```diff
@@ -47,31 +47,31 @@ Please include:
 
 ![#### App ID](//s3.amazonaws.com/user-content.stoplight.io/6012/1552664114224 "#### App ID")
 
-## Before Development
+## Before development
 
-BigCommerce will send requests to your server to get information back about shipping quotes, credential validation and configuration.
+BigCommerce will send requests to your server to get information back about shipping quotes, credential validation, and configuration.
 
-### Your Service URLs
+### Your service URLs
 Since BigCommerce will be sending requests, you will need to provide BigCommerce with URLs that can accept quote requests and optionally a URL to check and validate connection options during app registration. These can be any valid HTTPS URL using port `443`, for example:
 
 `https://yourhost.com/rate`
 
-Your host and rate should be replaced with your own host and path.
+Replace your host and rate with your own host and path.
 
 ### Routes
 
-You should create a URL to provide shipping quotes on your API. This is the URL used to check the available shipping rates. The second URL (optional) checks the merchant’s connection settings are valid. It can perform any checks necessary to do so such as looking up credentials in your database or calling a downstream service to verify them.
+You should create a URL to provide shipping quotes on your API and check the available shipping rates. The second URL (optional) checks to ensure the merchant’s connection settings are valid. It can perform any checks necessary to do so, such as looking up credentials in your database or calling a downstream service to verify them.
 
-### Requests and Responses
+### Requests and responses
 
-BigCommerce will send and receive data using JSON. The request for rates will always be formatted using the [Base Rate Request Model](https://developer.bigcommerce.com/api-reference/store-management/shipping-provider-api/shipping-provider/requestshippingrates). The response for rates should be formatted using the [Carrier Quote Object](https://developer.bigcommerce.com/api-reference/store-management/shipping-provider-api/shipping-provider/requestshippingrates). The request to check for merchant app credentials will be formatted as Check Connection Options Request Payload and the response should be formatted using the  Check Connection Options Response Payload.
+BigCommerce will send and receive data using JSON. The request for rates will always be formatted using the [Base Rate Request Model](https://developer.bigcommerce.com/api-reference/store-management/shipping-provider-api/shipping-provider/requestshippingrates). The response for rates should be formatted using the [Carrier Quote Object](https://developer.bigcommerce.com/api-reference/store-management/shipping-provider-api/shipping-provider/requestshippingrates). Format the request to check for merchant app credentials as Check Connection Options Request Payload, and the response should be formatted using the Check Connection Options Response Payload.

```

Comment:
The Base Rate Request Model link takes you to the Request Shipping Rates endpoint.